test-utils/client/src/lib.rs

```diff
@@ -255,3 +258,87 @@ impl<Block: BlockT, E, Backend, G: GenesisInit> TestClientBuilder<
 		self.build_with_executor(executor)
 	}
 }
+
+/// An extension trait for `RpcHandlers`.
+pub trait RpcHandlersExt {
```

Comment:
And this `SendTransaction`?

Reply:
It's good too. I think the trait can be extended for have more functionalities but since it doesn't have it at the moment. I'm happy both ways. What do you prefer?

Reply:
Yeah leave it.